src/Tizen.Multimedia.Camera/Camera/Camera.cs

```diff
@@ -69,12 +68,20 @@ public Camera(CameraDevice device)
         /// <summary>
         /// Initializes a new instance of the <see cref="Camera"/> class.
         /// </summary>
-        /// <remarks>CameraDevice and Type will be selected internally by CameraDeviceManager.</remarks>
-        /// <exception cref="InvalidOperationException">In case of any invalid operations.</exception>
+        /// <remarks>
+        /// If <see cref="CameraDeviceManager"/> is supported, <see cref="CameraDevice"/> will be selected automatically by internal policy.<br/>
+        /// (User can check whether <see cref="CameraDeviceManager"/> is supported or not by using <see cref="CameraDeviceManager.IsCameraDeviceManagerSupported"/>.)<br/>
```

Comment:
I would make a link (with `<see `) only for the first instance of the API item in the text, but it's not an error. No need to change anything here.

Reply:
I agree with you. I'll remove duplicated link after first one.